tests/e2e/vm_connectivity_test.go

```diff
@@ -0,0 +1,205 @@
+			return kubectl.RawCommand(subCMD, ShortWaitDuration)
+		}
+
+		ItApplyFromFile(vmOneIPClaim)
+		ItApplyFromFile(vmOneManifest)
+		ItApplyFromFile(vmSvcOne)
+		ItApplyFromFile(vmTwoIPClaim)
+		ItApplyFromFile(vmTwoManifest)
+		ItApplyFromFile(vmSvcTwo)
+
+		vmOne, err := GetVMFromManifest(vmOneManifest)
+		Expect(err).To(BeNil())
+		vmTwo, err := GetVMFromManifest(vmTwoManifest)
+		Expect(err).To(BeNil())
+
+		It(fmt.Sprintf("Wait %s running", vmOne.Name), func() {
+			waitVmStatus(vmOne.Name, VMStatusRunning)
+		})
+		It(fmt.Sprintf("Wait %s running", vmTwo.Name), func() {
+			waitVmStatus(vmTwo.Name, VMStatusRunning)
+		})
+		It("Wait 30 sec for sshd started", func() {
+			time.Sleep(30 * time.Second)
+		})
+
+		It("Check ssh via virtctl", func() {
+			command := "hostname"
+			CheckResultSshCommand(vmOne.Name, command, vmOne.Name)
+		})
+		It("Check external site from VM", func() {
+			command := "curl -o /dev/null -s -w \"%{http_code}\\n\" https://flant.com"
+			httpCode := "200"
+			CheckResultSshCommand(vmOne.Name, command, httpCode)
+		})
+
+		It(fmt.Sprintf("Get nginx page through service from %s", vmOne.Name), func() {
+			res := curlSVC(vmOne.Name, vmSvcOne, conf.Namespace)
+			Expect(res.Error()).NotTo(HaveOccurred(), "%s", res.StdErr())
+			Expect(strings.TrimSpace(res.StdOut())).Should(ContainSubstring(vmOne.Name))
+		})
+
+		It(fmt.Sprintf("Delete pod helper for %s", vmOne.Name), func() {
+			res := deletePodHelper(vmOne.Name, conf.Namespace)
+			Expect(res.Error()).NotTo(HaveOccurred(), "%s", res.StdErr())
+		})
+
+		It(fmt.Sprintf("Get nginx page through service %s", vmTwo.Name), func() {
+			res := curlSVC(vmTwo.Name, vmSvcTwo, conf.Namespace)
+			Expect(res.Error()).NotTo(HaveOccurred(), "%s", res.StdErr())
+			Expect(strings.TrimSpace(res.StdOut())).Should(ContainSubstring(vmTwo.Name))
+		})
+
+		It(fmt.Sprintf("Delete pod helper for %s", vmTwo.Name), func() {
+			res := deletePodHelper(vmTwo.Name, conf.Namespace)
+			Expect(res.Error()).NotTo(HaveOccurred(), "%s", res.StdErr())
+		})
+
+		It(fmt.Sprintf("Change label on %s", "vm1-service"), func() {
+			PatchSvcSelector := func(name, label string) {
+				GinkgoHelper()
+				PatchResource(kc.ResourceService, name, &kc.JsonPatch{
+					Op:    "replace",
+					Path:  "/spec/selector/service",
+					Value: label,
+				})
+			}
+
+			PatchSvcSelector("vm1-service", "v2")
+		})
+		It(fmt.Sprintf("Check label %s", "vm1-service"), func() {
+			GetSvcLabel := func(name, label string) {
+				GinkgoHelper()
+				output := "jsonpath={.spec.selector.service}"
+				CheckField(kc.ResourceService, name, output, label)
+			}
+			GetSvcLabel("vm1-service", "v2")
+		})
+
+		It(fmt.Sprintf("Get nginx page from %s and expect %s hostname", vmOne.Name, vmTwo.Name), func() {
+			res := curlSVC(vmOne.Name, vmSvcOne, conf.Namespace)
+			Expect(res.Error()).NotTo(HaveOccurred(), "%s", res.StdErr())
+			Expect(strings.TrimSpace(res.StdOut())).Should(ContainSubstring(vmTwo.Name))
+		})
+	})
+})
```

Comment:
I still think this is the not needed test because service work is contract with kubernetes